home/views.py

```diff
@@ -12,16 +12,29 @@ def landingpage(request):
     return render(request, 'home/landingpage.html')
 
 
-def displayQuestion(request, **kwargs):
-    question = get_object_or_404(Question, id=kwargs['pk'])
-    sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
+def display_question_page(request, pk):
+    def add_answer(form, profile, question):
```

Comment:
what is the point of having a nested function like this, it seems to be just adding some unnecessary complexity to the code...

Reply:
removed